src/ui/Facet/Facet.ts

```diff
@@ -1358,7 +1358,6 @@ export class Facet extends Component implements IFieldValueCompatibleFacet {
   protected updateAppearanceDependingOnState() {
     $$(this.element).toggleClass('coveo-active', this.values.hasSelectedOrExcludedValues());
     $$(this.element).toggleClass('coveo-facet-empty', !this.isAnyValueCurrentlyDisplayed());
-    $$(this.element).toggleClass('coveo-hidden', !this.getDisplayedFacetValues().length);
```

Comment:
This line was added as part of the dependent facet manager [refactor](https://github.com/coveo/search-ui/pull/1441/files#diff-75f83b041f6bf3f2237e38431ca3de03R1339), causing the breaking the change.

By removing this line and adding the one in the next file below, hiding and displaying a dependent facet now occurs entirely inside the manger.